repos/dir.props

```diff
@@ -46,6 +46,10 @@
     <EnvironmentVariables Include="DotNetBuildOffline=true" Condition="'$(OfflineBuild)' == 'true'" />
     <EnvironmentVariables Include="DotNetCoreSdkDir=$(DotNetCliToolDir)" />
     <EnvironmentVariables Include="DotNetBuildToolsDir=$(ProjectDir)Tools/" />
+    <EnvironmentVariables Include="StabilizePackageVersion=$(UseStableVersions)" Condition="'$(UseStableVersions)' != ''" />
+    <!-- for RTM only, default to stabilized builds because that's what matches the official builds -->
+    <EnvironmentVariables Include="StabilizePackageVersion=true" Condition="'$(UseStableVersions)' == ''" />
```

Comment:
It seems cleaner to me to default `UseStableVersions` itself in a property group.

Also, I filed this issue for stabilization config auto-update that you could link in a comment: https://github.com/dotnet/source-build/issues/550.

Reply:
Yeah I'd be OK with setting UseStableVersions and VersionStamp properties in the target/props file but that enables us to override them if necessary instead of having them be hard-coded as they originally where.